core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -348,9 +353,14 @@ private PutResult putAttempt(PageId pageId, byte[] page, CacheScope scope, Cache
           mPageStore.put(pageId, page);
           Metrics.BYTES_WRITTEN_CACHE.mark(page.length);
           return PutResult.OK;
+        } catch (ResourceExhaustedException e) {
+          undoAddPage(pageId);
+          LOG.error("Failed to add page {} to pageStore: ", pageId, e);
+          Metrics.PUT_STORE_WRITE_NO_SPACE_ERRORS.inc();
+          return PutResult.NO_SPACE;
         } catch (IOException e) {
           undoAddPage(pageId);
-          LOG.error("Failed to add page {}: {}", pageId, e);
+          LOG.error("Failed to add page {} to pageStore: ", pageId, e);
```

Comment:
Is the notation to have `: ` before the exception, I don't think we do this in other places?

Reply:
I was using this convention in this class